content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-composite-multifield-nodestore.js

```diff
@@ -22,89 +22,89 @@
-                                            sling:resourceType="granite/ui/components/foundation/form/fieldset"
-                                            acs-commons-nested="NODE_STORE"
-                                            name="./pages">
-                                                <layout
-                                                jcr:primaryType="nt:unstructured"
-                                                sling:resourceType="granite/ui/components/foundation/layouts/fixedcolumns"
-                                                method="absolute"/>
-                                                <items jcr:primaryType="nt:unstructured">
-                                                    <column
-                                                    jcr:primaryType="nt:unstructured"
-                                                    sling:resourceType="granite/ui/components/foundation/container">
-                                                        <items jcr:primaryType="nt:unstructured">
-                                                            <page
-                                                            jcr:primaryType="nt:unstructured"
-                                                            sling:resourceType="granite/ui/components/foundation/form/textfield"
-                                                            fieldDescription="Name of Page"
-                                                            fieldLabel="Page Name"
-                                                            name="./page"/>
-                                                            <path
-                                                            jcr:primaryType="nt:unstructured"
-                                                            sling:resourceType="granite/ui/components/foundation/form/pathbrowser"
-                                                            fieldDescription="Select Path"
-                                                            fieldLabel="Path"
-                                                            name="./path"
-                                                            rootPath="/content"/>
-                                                        </items>
-                                                    </column>
-                                                </items>
-                                            </field>
-                                        </pages>
-                                    </items>
-                                </column>
-                            </items>
-                        </fieldset>
-                    </items>
-                </column>
-            </items>
-        </content>
-    </jcr:root>
-</code>
-*/
+ <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
```

Comment:
apologies @stopp - notifications to @bstopp (read b stopp) are being sent to your account i believe (pls contact github)